plugin/src/main/resources/io/jenkins/plugins/remotingkafka/KafkaComputerLauncher/config.jelly

```diff
@@ -1,4 +1,13 @@
 <?jelly escape-by-default='true'?>
-<j:jelly xmlns:j="jelly:core"
->
-</j:jelly>
+<j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:l="/lib/layout"
+         xmlns:t="/lib/hudson" xmlns:f="/lib/form">
+    <f:entry title="${%Kafka username}" field="kafkaUsername">
```

Comment:
Need explanation why it is needed to evaluate

Reply:
@oleg-nenashev there might be different users allowed to access Kafka with respect to different usernames. Username in the global configuration is admin user.